backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtGraphics.java

```diff
@@ -339,7 +343,8 @@ private native void exitFullscreen () /*-{
 	@Override
 	public DisplayMode getDisplayMode () {
 		double density = config.usePhysicalPixels ? getNativeScreenDensity() : 1;
-		return new DisplayMode((int)(getScreenWidthJSNI() * density), (int)(getScreenHeightJSNI() * density), 60, 8) {};
+		return new DisplayMode((int)(getScreenWidthJSNI() * density), (int)(getScreenHeightJSNI() * density), 60,
+			getColorDepthJSNI()) {};
```

Comment:
This looks good, but is the color depth used anywhere in GWT? In any case, 8 appears to have been incorrect, since Chrome would have always returned 24 until a few years ago. Approving because this looks ,like it achieves the correct behavior; if there are issues with this, they're probably because something assumes 8bpp on GWT incorrectly.


...bits per pixle.

Reply:
Nothing in libGDX relies on this. On GWT, I mean, where it's impossible to change the desktop display mode.

It's possible a game might use this value - I've seen titles recommend or enforce a particular colour depth, all of which predate libGDX. But they'd be getting the wrong value on GWT. I haven't checked mobile!

Reply:
0fps and bpp on Android. Includes alpha on iOS. What a mess.

https://github.com/libgdx/libgdx/blob/79cf00af53b7f38667291fbacf544d3074a811bd/backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java#L690

https://github.com/libgdx/libgdx/blob/79cf00af53b7f38667291fbacf544d3074a811bd/backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java#L409-L410

Reply:
Oh, wow, that is a mess. So I suppose this means it doesn't matter currently, but could if > 24bpp monitors or mobile devices start becoming common.

Reply:
10-bit displays are already a thing for computers (no surprise there), Android and iOS. Don't know why you'd want to detect them, but may as well. Think I can I can improve the Android version....

Reply:
So I don't think there's a way of getting an actual bit depth from an Android device, but there is `Device#isWideColorGamut` and `#Display#isHdr`. Not of much use to us, but it's interesting to note. I really need to catch some Z's.